### DISCUSSION TITLE
Update to 3.23.157 won't launch on Linux Mint

### DISCUSSION THREAD

**Discussion by atanasentchev** [standardnotes/forum#2874] [General] [answered]
Getting message "Cannot mount Appimage, please check your FUSE setup." When I run with the suggested --appimage-extract, I get:

Something went wrong trying to read the squashfs image.
Failed to open squashfs image

Why is this happening, and how can I get past this and update and launch Standard Notes 3.23.157?

**Comment by adrihut** [comment, marked as answer]
I believe this has to do with SN needing the `libfuse2` package, and Ubuntu—_Mint's parent distro_—removing it, and moving to `libfuse3` instead as part of their latest release. Try installing **`libfuse2`** on your system by running the following command:

```
sudo apt install libfuse2
```
<br>

Hope this helps!

**Comment by atanasentchev** [comment]
Thank you, Adrian! This did indeed help. I am very impressed with your product and with your support.
